.tekton/collector-build.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
     # TODO(ROX-21073): re-enable for all PR branches
-    pipelinesascode.tekton.dev/on-cel-expression: (event == "push" && target_branch == "master") || (event == "pull_request" && (source_branch.contains("rhtap") || source_branch.contains("appstudio") || source_branch.contains("konflux")))
+    pipelinesascode.tekton.dev/on-cel-expression: (event == "push" && target_branch.matches("^(master|release-.*)$")) || (event == "pull_request" && source_branch.matches("(konflux|appstudio|rhtap)"))
```

Comment:
IIUC `matches` expects a regex to compare to the source branch here.

`(konflux|appstudio|rhtap)` would only match branches literally `konflux`, `appstudio`, `rhtap` - not `tm/konflux-something`.

Reply:
"matches" has different semantics in different languages. Here I assumed it checks that the string has a match, not that the entire string must match. What I assumed seems to be the case because the Konflux CI **runs** in this PR.

Just to be on the safe side and check that I did not mess up the condition so that it matches just any string, I tried with a "wrong" branch name https://github.com/stackrox/collector/pull/1777 and the Konflux CI does not run there.